Side-Projects.md

```diff
@@ -21,24 +21,30 @@ to your host computer's local disk
 
 ```bash
 cd <directory with docker-compose.yml>
-docker-compose up -d 
+docker compose up -d 
 ```
 
+Now as long as you don't `docker compose down`, the docker daemon (`sudo systemctl enable docker`) will automatically start the various containers every time the Pi boots.
```

Comment:
Do it need to be a 'Pi' or could it be just any host system?

Reply:
Fixed, thanks!